[Issue] CMake Error: `rocm_smi64 Target Includes Non-Existent Path` During Linux PyTorch `nightly` Wheels Build

**Platform:** `Linux`
**Pytorch Version:** `nightly`
**Python Versions:** `3.10, 3.11, 3.12, 3.13`

**Effected Archs:** `gfx1151, gfx950, gfx1150, gfx103x, gfx120x, gfx94x, gfx90a, gfx110, gfx101, gfx1152, gfx1153 `

### Runs:
- https://github.com/ROCm/TheRock/actions/runs/22572668776/job/65384176942
- https://github.com/ROCm/TheRock/actions/runs/22572142127/job/65382399064
- https://github.com/ROCm/TheRock/actions/runs/22570332110/job/65376281974
- https://github.com/ROCm/TheRock/actions/runs/22569799863/job/65374477141

```
2026-03-02T11:04:34.9857155Z --     USE_TENSORPIPE        : ON
2026-03-02T11:04:34.9857386Z --   Public Dependencies  : 
2026-03-02T11:04:34.9858722Z --   Private Dependencies : Threads::Threads;/opt/python/cp310-cp310/lib/python3.10/site-packages/_rocm_sdk_devel/lib/host-math/lib/librocm-openblas.so;pthreadpool;cpuinfo;pytorch_qnnpack;nnpack;XNNPACK;microkernels-prod;fbgemm;ittnotify;fp16;caffe2::openmp;tensorpipe;nlohmann;moodycamel;gloo;rt;fmt::fmt-header-only;kineto;gcc_s;gcc;dl
2026-03-02T11:04:34.9859979Z --   Public CUDA Deps.    : 
2026-03-02T11:04:34.9860206Z --   Private CUDA Deps.   : __caffe2_nccl
2026-03-02T11:04:34.9860449Z --   USE_COREML_DELEGATE     : OFF
2026-03-02T11:04:34.9860686Z --   BUILD_LAZY_TS_BACKEND   : ON
2026-03-02T11:04:34.9860904Z --   USE_ROCM_KERNEL_ASSERT : OFF
2026-03-02T11:04:34.9894939Z -- Configuring done (35.0s)
2026-03-02T11:04:36.5573742Z CMake Error in caffe2/CMakeLists.txt:
2026-03-02T11:04:36.5574241Z   Imported target "rocm_smi64" includes non-existent path
2026-03-02T11:04:36.5574771Z 
2026-03-02T11:04:36.5575502Z     "/home/runner/_work/TheRock/TheRock/output/build/third-party/sysdeps/linux/libdrm/build/stage/lib/rocm_sysdeps/lib/pkgconfig/../../include"
2026-03-02T11:04:36.5576201Z 
2026-03-02T11:04:36.5576424Z   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
2026-03-02T11:04:36.5576754Z 
2026-03-02T11:04:36.5576942Z   * The path was deleted, renamed, or moved to another location.
2026-03-02T11:04:36.5577269Z 
2026-03-02T11:04:36.5577467Z   * An install or uninstall procedure did not complete successfully.
2026-03-02T11:04:36.5577791Z 
2026-03-02T11:04:36.5578007Z   * The installation package was faulty and references files it does not
2026-03-02T11:04:36.5578359Z   provide.
2026-03-02T11:04:36.5578532Z 
2026-03-02T11:04:36.5578536Z 
2026-03-02T11:04:36.5579332Z 
2026-03-02T11:04:37.9952413Z CMake Warning:
2026-03-02T11:04:37.9952731Z -- Generating done (3.0s)
2026-03-02T11:04:37.9953186Z   Manually-specified variables were not used by the project:
2026-03-02T11:04:37.9953510Z 
2026-03-02T11:04:37.9953622Z     USE_FBGEMM_GENAI
2026-03-02T11:04:37.9953776Z 
2026-03-02T11:04:37.9953780Z 
2026-03-02T11:04:37.9954769Z CMake Generate step failed.  Build files cannot be regenerated correctly.
2026-03-02T11:04:38.1329393Z Traceback (most recent call last):
2026-03-02T11:04:38.1330223Z ++ Exec [/__w/TheRock/TheRock]$ /opt/python/cp310-cp310/bin/python -m pip cache remove rocm
2026-03-02T11:04:38.1330984Z   File "/__w/TheRock/TheRock/./external-builds/pytorch/build_prod_wheels.py", line 1262, in <module>
2026-03-02T11:04:38.1331772Z     main(sys.argv[1:])
2026-03-02T11:04:38.1332489Z   File "/__w/TheRock/TheRock/./external-builds/pytorch/build_prod_wheels.py", line 1258, in main
2026-03-02T11:04:38.1333048Z     args.func(args)
2026-03-02T11:04:38.1333715Z   File "/__w/TheRock/TheRock/./external-builds/pytorch/build_prod_wheels.py", line 622, in do_build
2026-03-02T11:04:38.1334633Z Warning: pip cache remove failed (cache may be disabled), continuing
2026-03-02T11:04:38.1335646Z ++ Exec [/__w/TheRock/TheRock]$ /opt/python/cp310-cp310/bin/python -m pip install --force-reinstall --pre --index-url https://rocm.nightlies.amd.com/v2/gfx120X-all/ 'rocm[libraries,devel]==7.12.0a20260302'
2026-03-02T11:04:38.1337216Z ++ Capture [/__w/TheRock/TheRock]$ /opt/python/cp310-cp310/bin/python -m rocm_sdk version
2026-03-02T11:04:38.1337690Z Installed version: 7.12.0a20260302
```

--- a/projects/rocm-smi-lib/rocm_smi/CMakeLists.txt
+++ b/projects/rocm-smi-lib/rocm_smi/CMakeLists.txt
@@ -94,8 +94,6 @@ target_include_directories(${ROCM_SMI_TARGET}
                            "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
 )
 
-target_include_directories(${ROCM_SMI_TARGET} INTERFACE ${DRM_INCLUDE_DIRS})
-
 ## Set the VERSION and SOVERSION values
 set_property(TARGET ${ROCM_SMI_TARGET} PROPERTY
              SOVERSION "${VERSION_MAJOR}")

